.github/workflows/tests.yaml

```diff
@@ -160,7 +160,7 @@ jobs:
           # When running directly against Firefox, we test only using geckodriver (not against legacy Firefox =<45), so we must declare GECKODRIVER=1
           GECKODRIVER: "${{ (matrix.browser == 'firefox' && !matrix.selenium-server) && '1' || '0' }}"
           # Provide CHROMEDRIVER_PATH and GECKODRIVER_PATH so that tests for local web drivers are able to start the browser
-          CHROMEDRIVER_PATH: "${{ (matrix.browser == 'chrome' && !matrix.selenium-server) && '/usr/local/share/chrome_driver/chromedriver' || '' }}"
+          CHROMEDRIVER_PATH: "${{ (matrix.browser == 'chrome' && !matrix.selenium-server) && '/usr/local/share/chromedriver-linux64/chromedriver' || '' }}"
```

Comment:
can we use `command chromedriver` or `which chromedriver`?

we can run separate step, detect location of bin and use it in another step like this.

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
Yes, but this would mean rewriting this whole section to `run` script, as you unfortunately cannot use values of other environment variables inside `env` section. So even if we save the path to environment variable using `which`, it would not be possible to use the value here...